mmv1/templates/terraform/examples/cloudrun_vpc_access_connector.tf.erb

```diff
@@ -0,0 +1,73 @@
+# [START vpc_serverless_connector_enable_api]
+resource "google_project_service" "vpcaccess_api" {
```

Comment:
Each resource in this config will need to be annotated with the `google-beta` provider as this test is present on a beta resource.

```suggestion
resource "google_project_service" "vpcaccess_api" {
  provider = google-beta
```

Reply:
@betsy-lichtenberg: Note that this'll need to be applied to each resource, not just this one!